salt/search/config/lib-systemd-system-gearman-job-server.service

```diff
@@ -8,7 +8,7 @@ User=gearman
 Restart=always
 PIDFile=/run/gearman/server.pid
 ExecStart=/usr/sbin/gearmand \
-    --listen=127.0.0.1 \
+    --listen=localhost \
```

Comment:
http://gearman.org/manual/job_server/#options seems to say we could just remove it because the default is "any"? Untested

Reply:
huh - '127.0.0.1' is the default that came in the service file.

I'll merge this, try it on CI (working) and end2end (not working) and see what happens